src/Components/Register/Register.js

```diff
@@ -21,26 +21,52 @@ const Register = ({ onSubmit, errorLabel }) => {
     password: '',
     repeatPassword: '',
     alterEgo: '',
-    icon: '' });
+    icon: '',
+    isPwdMatched: false
+  });
 
   const handleChange = ({ target }) => {
     const { name, value } = target;
+    let algo = {};
+    console.log('Target', name, value, target);
+    if(name === 'password'){
+        if(value === payload.repeatPassword && value !== ''){
+          algo.isPwdMatched = true;
+        }else{
```

Comment:
Please add spaces to make it more legible

Reply:
Done.